test/e2e/webhook_test.go

```diff
@@ -93,6 +161,47 @@ var _ = Describe("Workspace Validation Webhook", func() {
 
 	})
 
+	It("should validate the workspace tuning spec at update ", func() {
+		workspaceObj := utils.GenerateTuningWorkspaceManifest(fmt.Sprint("webhook-", rand.Intn(1000)), namespaceName, "", 1, "Standard_Bad",
+			&metav1.LabelSelector{
+				MatchLabels: map[string]string{"kaito-workspace": "webhook-e2e-test"},
+			}, nil, validInput, validOutput, validPreset, validMethod)
+
+		By("Creating a valid tuning workspace", func() {
```

Comment:
why do we need this one in update test?

Reply:
@Fei-Guo Yes, it is duplicate, just updated